modules/sql-engine/src/integrationTest/sql/basic_queries/not_supported_queries.test

```diff
@@ -0,0 +1,5 @@
+# name: test/sql/basic_queries/not_supported_queries.test
+# group: [basic_queries]
+
+statement error: Failed to validate query. From line 1, column 6 to line 1, column 14: A recursive query is not supported.
```

Comment:
Let's add mention SQL Standart features T131(Recursive query) and T132 (Recursive query in subquery)
Also, we need to add statement for the T132 feature.

Reply:
done